src/CMakeLists.txt

```diff
@@ -106,6 +106,7 @@ target_link_libraries(osp-magnum PRIVATE
     MagnumPlugins::TinyGltfImporter
     MagnumPlugins::StbImageImporter
     dNewton dScene dModel dVehicle
+    toml11
```

Comment:
Did this end up being needed to build on windows? 

I'm a bit confused, cause the windows continuous integration is working. What happened?

Reply:
I accidently added that, removed it just now